meta-lmp-support/recipes-kernel/linux/linux-lmp-fslc-imx_git.bbappend

```diff
@@ -1,3 +1,3 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-SRC_URI_append_imx8mmevk += "file://0001-Disable-1V8-on-usdhc2-for-SDMUX-kernel.patch "
+SRC_URI_append_imx8mm-lpddr4-evk = " file://0001-Disable-1V8-on-usdhc2-for-SDMUX-kernel.patch"
```

Comment:
this is now different target than previously imx8mmevk vs imx8mm-lpddr4-evk

Reply:
Foundries seem to have added recipe still to enable use of old name (imx8mmevk) though, but I think we should use this for now on as this is the official name.